client/state/data-layer/wpcom/sites/media/index.js

```diff
@@ -42,13 +42,22 @@ export function requestMedia( action ) {
 	const path =
 		query && query.source ? `/meta/external-media/${ query.source }` : `/sites/${ siteId }/media`;
 
+	const googlePhotosPicker = query.source === 'google_photos' && query.session_id;
+	const fetchOptions = googlePhotosPicker
+		? {
+				apiNamespace: 'wpcom/v2',
+		  }
+		: {
+				apiVersion: '1.1',
```

Comment:
Is `apiVersion` needed here if we're calling `wpcom/v2`?

Reply:
I covered a new flow integration with this change, but the existing media request remained the same.
See: https://github.com/Automattic/wp-calypso/blob/trunk/client/state/data-layer/wpcom/sites/media/index.js#L50

Reply:
I somehow missed the condition check here...sorry!